ompi/communicator/comm_cid.c

```diff
@@ -907,9 +909,8 @@ int ompi_comm_activate_nb (ompi_communicator_t **newcomm, ompi_communicator_t *c
      * 1. The communicator's disjointness is inferred from max_local_peers.
      * 2. After the operation it is allowed to send messages over the new communicator.
      */
-    local_peers = context->max_local_peers;
-    ret = context->allreduce_fn (&local_peers, &context->max_local_peers, 1, MPI_MAX, context,
-                                 &subreq);
+    ret = context->iallreduce_fn (&context->local_peers, &context->max_local_peers, 1, MPI_MAX, context,
```

Comment:
Need to delete `local_peers` declaration

Reply:
on line 910 or 271?

Reply:
at the beginning of this function where `int local_peers = -1;`